examples/bank_reserves/batch_run.py

```diff
@@ -181,25 +181,36 @@ def run_model(self):
 
 # parameter lists for each parameter to be tested in batch run
 br_params = {
-    "init_people": [25, 100, 150, 200],
-    "rich_threshold": [5, 10, 15, 20],
-    "reserve_percent": [0, 50, 100],
+    "init_people": [25, 100],
+    "rich_threshold": [5, 10],
+    "reserve_percent": 5,
 }
 
-br = BatchRunner(
+"""br = BatchRunner(
     BankReservesModel,
     br_params,
-    iterations=1,
+    iterations=2,
     max_steps=1000,
-    model_reporters={"Data Collector": lambda m: m.datacollector},
-)
+    nr_processes=None,
+    # model_reporters={"Data Collector": lambda m: m.datacollector},
+)"""
 
 if __name__ == "__main__":
-    br.run_all()
+    data = batch_run(
+        BankReservesModel,
+        br_params,
+        model_reporters={"Rich": get_num_rich_agents},
+        agent_reporters={"Wealth": "wealth"},
+    )
+    br_df = pd.DataFrame(data)
+    br_df.to_csv("BankReservesModel_Data.csv")
+    # br.run_all()
+    """
```

Comment:
The docstring-ed code below should probably be removed?

Reply:
@rht I want to push this PR thru. Can you either fix this in another PR or create a ticket?

Reply:
I made #1110.